_posts/2025-05-20-firefox-security-privacy-newsletter-2025-q1.md

```diff
@@ -0,0 +1,72 @@
+# Firefox Security and Privacy Newsletter 2025 Q1
+
+Security and Privacy on the web are the cornerstones of [Mozilla’s manifesto](https://www.mozilla.org/en-US/about/manifesto/), and they influence how we operate and build our products. Following are the highlights of our work from Q1 2025, grouped into the following categories:
+
+* **Firefox Product Security & Privacy**, showcasing new Security & Privacy Features and Integrations in Firefox.  
+* **Core Security**, outlining Security and Hardening efforts within the Firefox Platform.  
+* **Web Security**, allowing websites to better protect themselves against online threats.
+
+## Preface
+
+Note: Some of the bugs linked below might not be accessible to the general public and are still restricted to specific work groups. [We de-restrict fixed security bugs after a grace-period](https://firefox-source-docs.mozilla.org/bug-mgmt/processes/fixing-security-bugs.html#keeping-private-information-private), until the majority of our user population have received their updates. If a link does not work for you, please accept this as a precaution for the safety of all of our users.
```

Comment:
I think it's better to remove this. Afaict all the bugs are open and nobody wants to read Kleingedrucktes.